eng/pipelines/runtimelab.yml

```diff
@@ -58,21 +58,21 @@ stages:
   - template: /eng/pipelines/common/checkout-job.yml
 
   #
-  # Build with Debug config and Checked runtimeConfiguration
+  # Build with Release config and Debug runtimeConfiguration
   #
   - ${{ if ne(variables.isOfficialBuild, true) }}:
     - template: /eng/pipelines/common/platform-matrix.yml
       parameters:
         jobTemplate: /eng/pipelines/common/global-build-job.yml
-        buildConfig: Checked
+        buildConfig: Debug
         platforms:
         - Linux_x64
         - windows_x64
         - OSX_x64
         jobParameters:
           timeoutInMinutes: 90
           testGroup: innerloop
-          buildArgs: -s nativeaot+libs+nativeaot.packages -lc release -rc checked
+          buildArgs: -s nativeaot+libs+nativeaot.packages -lc release -rc debug
           extraStepsTemplate: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
 
   #
```

Comment:
Would it make sense this other run to be Release config with Checked runtime configuration?

Reply:
I think it is useful to run RyuJIT with both optimizations and asserts enables. It caches most of the JIT bugs.

Reply:
Yep, that makes sense. There's an assertion failure - I'm fixing that in another pull request.

We don't run with assertions in CHK builds because dotnet/runtime#43017 was still not merged :/